coderd/database/dbgen/dbgen.go

```diff
@@ -893,6 +894,36 @@ func CustomRole(t testing.TB, db database.Store, seed database.CustomRole) datab
 	return role
 }
 
+func CryptoKey(t testing.TB, db database.Store, seed database.CryptoKey) database.CryptoKey {
+	t.Helper()
+
+	b := make([]byte, 96)
```

Comment:
These lengths vary by feature so might want to let this be configurable somehow

Reply:
Isn't that what the `seed` is for?

Reply:
Sure, but everywhere we need to generate a different sized key means we have to generate the key separately to this helper

Reply:
Ah I see what you mean. Sure I can add a little switch statement.